tests/Unit/Account/Session/Oauth2/ShopSession/RefreshTokenTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace PrestaShop\Module\PsAccounts\Tests\Unit\Account\Session\Oauth2\ShopSession;
+
+use PrestaShop\Module\PsAccounts\Account\Session\ShopSession;
+use PrestaShop\Module\PsAccounts\Account\Token\Token;
+use PrestaShop\Module\PsAccounts\Exception\RefreshTokenException;
+use PrestaShop\Module\PsAccounts\Repository\ShopTokenRepository;
+use PrestaShop\Module\PsAccounts\Tests\TestCase;
+use PrestaShop\Module\PsAccounts\Vendor\League\OAuth2\Client\Token\AccessToken;
+
+class RefreshTokenTest extends TestCase
+{
+    use \PrestaShop\Module\PsAccounts\Tests\Unit\Account\Session\SessionHelpers;
+
+    /**
+     * @inject
+     *
+     * @var ShopTokenRepository
+     */
+    protected $repository;
```

Comment:
the test should not rely on deprecated services

Reply:
Actually, i have to check if the token is still present in the db after the test is done.
What's your idea on how to check without the repository?

Reply:
`Firebase\ShopSession` is the replacement